### DISCUSSION TITLE
Due diligence audit

### DISCUSSION THREAD

**Discussion by FraserMuir** [excalidraw/excalidraw#4787] [Q&A] [answered]
We use the free version of excalidraw at work however I have recently advocated for the paid teams plan. In order to support this tool our company must conduct a due diligence audit such as data protection, ownership and infrastructure. I've had a look on the website's privacy policy but it doesn't seem to cover much around how data is stored for excalidraw plus.

I understand you use the Google Storage API / Firebase, but where physically is the data stored - i.e. what region of the world? It's important for my company to understand since this could be used to store sensitive customer information - although we don't explicitly plan on it, it's a whiteboard so these things can occur - so we must ensure that the security practises are compliant to the same standards we are (we're based in the UK)

**Comment by dwelle** [comment, marked as answer]
Hey @FraserMuir. We do use Google Firebase at the moment, yes. The data is stored in the US. Feel free to contact us on support@excalidraw.com to discuss this further.
